GeneralsMD/Code/GameEngine/Source/Common/CRCDebug.cpp

```diff
@@ -184,9 +232,47 @@ void addCRCDebugLine(const char *fmt, ...)
 	}
 }
 
+void addCRCDebugLineNoCounter(const char *str)
```

Comment:
This function looks similar to `addCRCDebugLine`. Perhaps it can be merged with `addCRCDebugLine` to provide both variants in one? Or `addCRCDebugLine` can be split into smaller functions that can be reused on both function variants?